### PR TITLE
[redfish] Modify http response codes

### PR DESCRIPTION
**Pull request by zyoung51**

### CODE REVIEW

**Comment by zyoung51** [comment]
Depends on: https://github.com/RackHD/RackHD/pull/433

File: lib/api/redfish-1.0/event-service.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ var testEvent = controller(function(req, res) {
         });
     })
     .then(function() {
-        res.status(200);
+        res.status(202);
```
Comment (zyoung51) [inline lib/api/redfish-1.0/event-service.js]: Good catch!  The unit test was still checking for 200, and this was still returning 200.  Updated response code in controller flags and corresponding unit test.

**Comment by rackhdmirror** [comment]
Build finished.

**Comment by jlongever** [comment]
Jenkins: Depends on: RackHD/RackHD#433
test this please

**Comment by jlongever** [comment]
Dependency PR build not working.  Merging.

**Pull request merged**